### DISCUSSION TITLE
Is BSD support going to be dropped?

### DISCUSSION THREAD

**Discussion by Vringe** [vercel/community#6767] [Help]
### Summary

Our infrastructure runs almost entirely on FreeBSD. We have a lot of customers using NextJS. It is getting harder and harder to successfully build applications made with NextJS.

Here, someone stated, that the FreeBSD support has been deprecated
https://github.com/vercel/next.js/issues/59997

I would like to know why this decision was made.

### Steps to Reproduce

-

**Comment by amyegan** [comment]
Hey @Vringe. A fix was released in `14.2.1-canary.6` to solve related issue https://github.com/vercel/next.js/issues/64468. If you haven't already, please give that version a try and let me know whether it works for you

**Reply by Vringe** [reply]
This looks familiar. When running `npm run build` it just crashes at `Creating an optimized production build ..` without any error.

This also happens with `14.2.1-canary.6`. I just tried it.
